### ISSUE TITLE
'spec/fixtures' is hardcoded

### ISSUE THREAD

**Issue by kobrigo** [billtrik/karma-fixture#3]
The 'spec/fixtures' base path is actually hardcoded into the karma plugin.
On the adapter.js there is a call to Fixture "constructor" without paramters which makes it always use the 'spec/fixtures' as the prefix for all the paths of my fixtures (on the retrieval).
This requires that you save the files in a specific place under the karama base path (spec/fixtures).

On retrieval it checks for **html**["" + this.base + "/" + filename]
It would be nice to have the base path be configured from a parameter. or just remove this hardcoded prefix.

Thanks in advance.

**Comment by mklaber** [comment]
:+1: 

Ugly work around is to set the `base` property on `fixture`:

``` javascript
  beforeEach(function() {
    fixture.base = 'test/fixtures';
    var fixtures = fixture.load('test-fixture.json');
...
```

Gross but it should work until this gets properly resolved.

**Comment by billtrik** [comment]
@kobrigo @mklaber Unfortunately i cannot find any programmatic way to pass config options to the plugin inside the context iframe.

The only way i can think of is a permutation of @mklaber's way. 
You add the one of the following to a setup file that is included and always executed before your specs:

``` javascript
window.fixture = new Fixture('your_path_to_your_fixtures')
// or 
window.fixture.base = 'your_path_to_your_fixtures'
```

Let me know if you have any other suggestions.

**Comment by billtrik** [comment]
I published [0.2.3](https://github.com/billtrik/karma-fixture/tree/v0.2.3), where i created a `.setBase()` method that can be executed once in a top level `before()`.

One other option is to create a spec setup file, and load it before your specs, similarly to the require.js config file. Inside that file you could run `fixture.setBase(my/fixtures/base)`.

I also implemented loading fixtures by passing their full filenames, including their base path, thus overriding the `fixture.base` value.

I could not find a way to pass configuration from the `karma.conf.js` file inside the context.html runtime, other that writing a preprocessor plugin, and rewriting the html2js plugin for this purpose seems like overkill. If you find/know any other way please let me know.
